### PR TITLE
Add mandatory metadata in LogEntry if exists.

### PR DESCRIPTION
**Pull request by siddharthck**
Add static metadata _integration : azure .
Add event metadata for following keys :
level -> severity, operationName -> activity_type, resourceId -> azure_resource_id, category -> category .

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,6 +35,9 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:4.9.3'
     implementation 'io.gsonfire:gson-fire:1.8.5'
     implementation 'org.slf4j:slf4j-api:1.7.25'
+    implementation 'org.apache.commons:commons-lang3:3.9'
+    implementation 'com.jayway.jsonpath:json-path:2.5.0'
```
Comment (siddharthck) [inline build.gradle:39]: We are already using jayway jsonpath library in many internal projects. It has Apache 2 license. : 
https://github.com/json-path/JsonPath

**Comment by choudharypooja** [comment]
Please add the local.settings.json file as well

**Review by vishaalcharpe — APPROVED** [review]
Looks good to me!

**Review by choudharypooja — CHANGES_REQUESTED** [review]
local.settings.json missing still i guess

**Review by vishaalcharpe — CHANGES_REQUESTED** [review]
Please handle changes mentioned by Pooja

**Review by choudharypooja — APPROVED** [review]

**Comment by choudharypooja** [comment]
LGTM

**Review by mukundneharkar — APPROVED** [review]

**Review by vishaalcharpe — COMMENTED** [review]
Approved!

**Pull request merged**